packages/client/src/workflow-client.ts

```diff
@@ -268,6 +299,31 @@ export interface AsyncWorkflowListIterable extends AsyncIterable<WorkflowExecuti
   intoHistories: (intoHistoriesOptions?: IntoHistoriesOptions) => AsyncIterable<HistoryAndWorkflowId>;
 }
 
+/**
+ * A client-side handle to an Update.
+ */
+export interface WorkflowUpdateHandle<Ret> {
+  /**
+   * The ID of this Update request.
+   */
+  id: string;
```

Comment:
Is it really pertinent to expose the request id on the handle? If it is, then maybe we should qualify its name, something like `updateRequestId`. My concern is that, since we're not making this mandatory, most users will like not be "familliar" with the meaning of that "id".

Reply:
I personally like the shorter `id`, and figure folks will learn from the apidoc and become familiar.

Reply:
> Is it really pertinent to expose the request id on the handle?
Yes, this is the update ID, not just the request ID. And it's very important for those starting but not setting an explicit ID to have the value they can later reference elsewhere for recreating the handle. This is just like workflow ID for workflow handles.

> If it is, then maybe we should qualify its name, something like updateRequestId.

:+1: Looks like @dandavison set this to `updateId` (correctly IMO)

> I personally like the shorter id, and figure folks will learn from the apidoc and become familiar.

This appears to be already settled ground in the TypeScript SDK, i.e. `WorkflowHandle` has `workflowId` not just `id`, so `WorkflowUpdateHandle` should have `updateId` not `id` (as it now does, which I think is correct)